### PR TITLE
(post) - 04/06/2020 IAC Blog Post

### PR DESCRIPTION
**Pull request by david22swan**

### CODE REVIEW

File: _posts/2020-06-05-status-update.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - modules
+---
+
+Greetings from the IAC Team - here's our status update for this week!
+
+## Temporary Interuption to Test Infrastructure
+Due to the AddTrust intermediate certificate expiring, all Github Actions tests that we are running to verify modules on commercially licensed and/or non-dockerizable OS, we were not running any releases this week. A support ticket with VMware has been raised as apparently changing certificate chains is not as straight-forward as one might expect. Big Thanks to the DIO team who is working tirelessly on this issue.
+
+## Module Releases from IAC Team
+The release process for the following modules has been started and although we have had some issues, as was elaborated on above, we hope t get them out soon.
```
Comment (pmcmaw) [inline _posts/2020-06-05-status-update.md:30]: t -> to

File: _posts/2020-06-05-status-update.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- [puppetlabs/postgresql](https://github.com/puppetlabs/puppetlabs-postgresql) (6.6.0)
+- [puppetlabs/ibm_installation_manager](https://github.com/puppetlabs/puppetlabs-ibm_installation_manager) (2.2.0)
+
+## Community Contributions
+The wonderful community members that continue to contribute to our modules have provided us with the following enhancements:
+- **[`puppetlabs-java_ks#319`](https://github.com/puppetlabs/puppetlabs-java_ks/pull/319):** [tomkitchen](https://github.com/tomkitchen) a first time contributor has worked hard to add the ability to import a der formatted certificate file and it's corresponding key into a keystore. 
+
+Many thanks to [tomkitchen](https://github.com/tomkitchen) for the work that he has put n and we hope to see more in the future.
```
Comment (pmcmaw) [inline _posts/2020-06-05-status-update.md:38]: n -> in

File: _posts/2020-06-05-status-update.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Ubuntu 20.04 Certification
+Over the last week or two [I](https://github.com/david22swan) have personally worked on adding support to all of our relevant modules with much of it having been completed. As of right now there are only a small handfull of modules that require some futher work in order for us to officialy declare that the support the new OS.
+
+## New Litmus Github IO Page
+A new docs microsite containing all that we know about the excellent Litmus tool has been released just this week. This [site]({% link https://puppetlabs.github.io/litmus %}), created with care by the combined brilliance of our very own [David Schmidt](https://github.com/DavidS) and the docs teams star [Claire Cadman](https://github.com/clairecadman) will take you through everything you need to know about Litmus and help you get of the ground running with a mix of technical documentation and ste-by-step walkthroughs.
```
Comment (pmcmaw) [inline _posts/2020-06-05-status-update.md:44]: ste-by-step -> step-by-step

File: _posts/2020-06-05-status-update.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "29th May 2020: IAC Team Status Update"
```
Comment (pmcmaw) [inline _posts/2020-06-05-status-update.md:3]: 5th June

File: _posts/2020-06-05-status-update.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+author: david22swan
+categories:
+  - team
+  - status
+tags:
```
Comment (pmcmaw) [inline _posts/2020-06-05-status-update.md:8]: I think quite a few of the tags aren't relevant this week?

File: _posts/2020-06-05-status-update.md (hunk 3 above)
Comment (sanfrancrisko) [reply _posts/2020-06-05-status-update.md:44]:
```suggestion
A new docs microsite containing all that we know about the excellent Litmus tool has been released just this week. This [site]({% link https://puppetlabs.github.io/litmus %}), created with care by the combined brilliance of our very own [David Schmidt](https://github.com/DavidS) and the docs teams star [Claire Cadman](https://github.com/clairecadman) will take you through everything you need to know about Litmus and help you get of the ground running with a mix of technical documentation and step-by-step walkthroughs.
```

File: _posts/2020-06-05-status-update.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+- [puppetlabs/ibm_installation_manager](https://github.com/puppetlabs/puppetlabs-ibm_installation_manager) (2.2.0)
+
+## Community Contributions
+The wonderful community members that continue to contribute to our modules have provided us with the following enhancements:
+- **[`puppetlabs-java_ks#319`](https://github.com/puppetlabs/puppetlabs-java_ks/pull/319):** [tomkitchen](https://github.com/tomkitchen) a first time contributor has worked hard to add the ability to import a der formatted certificate file and it's corresponding key into a keystore. 
+
+Many thanks to [tomkitchen](https://github.com/tomkitchen) for the work that he has put n and we hope to see more in the future.
+
+## Ubuntu 20.04 Certification
+Over the last week or two [I](https://github.com/david22swan) have personally worked on adding support to all of our relevant modules with much of it having been completed. As of right now there are only a small handfull of modules that require some futher work in order for us to officialy declare that the support the new OS.
```
Comment (sanfrancrisko) [inline _posts/2020-06-05-status-update.md:41]:
```suggestion
Over the last week or two [I](https://github.com/david22swan) have personally worked on adding support to all of our relevant modules with much of it having been completed. As of right now there are only a small handful of modules that require some further work in order for us to officially declare that the support the new OS.
```

**Comment by DavidS** [comment]
accidentally force pushed over @david22swan 's changes. Reverted.

**Pull request merged**